obstacle_stop_planner_refine/include/obstacle_stop_planner/util.hpp

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef OBSTACLE_STOP_PLANNER__UTIL_HPP_
+#define OBSTACLE_STOP_PLANNER__UTIL_HPP_
+
+#include <string>
+
+#include "opencv2/core/core.hpp"
+#include "geometry_msgs/msg/point.hpp"
+#include "boost/geometry.hpp"
+#include "boost/format.hpp"
+#include "boost/assign/list_of.hpp"
+#include "tf2/utils.h"
+#include "diagnostic_msgs/msg/diagnostic_status.hpp"
+#include "diagnostic_msgs/msg/key_value.hpp"
+
+namespace bg = boost::geometry;
+using Point = bg::model::d2::point_xy<double>;
+using Polygon = bg::model::polygon<Point, false>;  // clockwise = false
+using Line = bg::model::linestring<Point>;
```

Comment:
These geometry types can be replaced by autoware_utils.

Reply:
OK. af960be

Reply:
Typing `autoware_utils::Point2d` every time is a bit bothering, so I recommend using `using autoware_utils::Point2d;`.

Reply:
Thanks! Modified in 3955aef